src/workers/notifications/postBookmarkReminder.ts

```diff
@@ -0,0 +1,39 @@
+import { messageToJson } from '../worker';
+import { Bookmark } from '../../entity';
+import { NotificationType } from '../../notifications/common';
+import { NotificationWorker } from './worker';
+import { buildPostContext } from './utils';
+
+interface Data {
+  postId: string;
+  userId: string;
+}
+
+const worker: NotificationWorker = {
```

Comment:
Can you use `TypedWorker` please? You can see examples in some recent ones like `commentUpvoteCanceledRep`. If you need any support on it let me know.

Reply:
Oh, cool. Let me check that now 👀

Reply:
Do we have a `TypedWorker` for notifications-related workers? The type doesn't seem to work when applied.